apps/lfx-one/src/app/shared/components/tag/tag.component.html

```
@@ -1,8 +1,20 @@
 <!-- Copyright The Linux Foundation and each contributor to LFX. -->
 <!-- SPDX-License-Identifier: MIT -->
 
-<p-tag [value]="value()" [severity]="severity()" [icon]="dot() ? undefined : icon()" [rounded]="rounded()" [styleClass]="styleClass()">
-  @if (dot()) {
-    <span class="inline-block h-1.5 w-1.5 shrink-0 rounded-full bg-current" aria-hidden="true"></span>
-  }
-</p-tag>
+@if (outlined()) {
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)